mockspec/generate.go

```diff
@@ -16,6 +17,10 @@ const (
 	commentPrefix = "#"
 )
 
+var (
+	singlePackage = flag.String("only-package", "", "Only generate mocks for the given package, ie: the first part of the line in the rules file")
```

Comment:
If it's the package name, rename the variable as such.
```suggestion
	packageName = flag.String("packageName", "", "Only generate mocks for the given package, ie: the first part of the line in the rules file")
```